src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      class="form"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        class="search search__big"
+        name="bigBox"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form
+      action="#"
+      method="get"
+      class="form form--small"
+      data-qa="small"
+    >
+      <input
+        type="text"
+        class="search search__small"
```

Comment:
And here you will modify your base class loop
In this way you do not need to repeat yourself by using the same styles as for search__small and search__big
You will set common styles for .loop, and ONLY styles that makes difference to loop--small
```suggestion
        class="search loop loop--small"
```